lib/updaters/index.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path')
+const JSON_BUMP_FILES = require('../../defaults').bumpFiles
+const PLAIN_TEXT_BUMP_FILES = ['VERSION.txt', 'version.txt']
+
+function getUpdaterByType (type) {
+  try {
+    return require(`./types/${type}`)
+  } catch (e) {
+    throw Error(`Unable to locate updated for provided type (${type}).`)
+  }
+}
+
+module.exports.getUpdaterByType = getUpdaterByType
+
+module.exports.getUpdaterByFilename = function (filename) {
+  if (JSON_BUMP_FILES.includes(filename)) {
```

Comment:
```suggestion
  if (JSON_BUMP_FILES.includes(path.basename(filename))) {
```

@jbottigliero You can use the same updater if the file is in a directory. To avoid this configuration:

```json
"bumpFiles": [
    {
        "filename": "src/manifest.json",
        "type": "json"
    }
]
```